omero/sysadmins/index.rst

```diff
@@ -101,7 +143,6 @@ OMERO clients to host images for groups or public viewing.
     :maxdepth: 2
     :titlesonly:
 
-    customization
```

Comment:
The customization page was placed here deliberately when this section was picked out to target facility managers - the rename is fine but I don't think we should move it away from this section.

Reply:
it is not really a server maintenance so I am not convinced it was right in the first place

Reply:
It was under "Optimizing OMERO as a Data Repository" with the intro text that specifically refers to customizing the appearance of the client

Reply:
humm we have several customization omero web pages that it is actually very confusing to find the correct one: customising appearance, customisation for caching and prefix, list of useful options => a bit of a mess

Something to tackle in a second round when we split the settings.py as suggested by @sbesson during a recent call.

Reply:
Yeah, there's a customizing section on what's currently called the OMERO.web administration page too

Reply:
I think having "OMERO.web UI customization" link under the "Optimizing OMERO as a Data Repository" heading is a bit misleading since almost none of those customisations are limited to a Public use-case but it's likely they will be missed by users not using Public groups.
But happy for this to be discussed & addressed later (not a blocker).

Reply:
A data repository doesn't have to be public though.

We can revisit the various customization sections after 5.4.0, we just might need to update the links on the website.

Reply:
@will-moore I agree with you that's why I initially moved it but after discussion with @hflynn we opted not to move so the change does not break the website
as mentioned previously web-customization needs to be reviewed, it is very confusing at the moment and even more depending on the way it is installed i.e. ``OMERO.server/bin/omero vs ``OMERO.py/bin/omero``

We decided to tackle web options  in another round